academy/ibc/ibc-token-lifecycle.md

```diff
@@ -0,0 +1,61 @@
+
+Therefore, in systems where Oracles are relied upon for a critical input, the trustworthiness of the overall system is no greater than trust in the Oracle. Consequently, such systems tend to focus on designs that encourage strong trust in the Oracle. Generally speaking, bridges are, in part, Oracles that can observe both chains as well in sign transactions and inject information about observations into the other chain. 
+
+Another challenge arises from differences in the way two chains operate. They may have differring block times. Even more importantly, one chain may be *probablistic* while the other is *deterministic*. Consider the following. If chain A releases tokens based on something that happened on chain B and then chain B reverted the original transaction (because chain B is probalistic and this possibility can never be completely eliminated), then a double-spend vulnerability emerges.
+
+## Design constraints
+
+There are obvious usabililty requirements for a successful bridge:
+
+1. Easy to use
+2. Innexpensive
+3. Reasonably fast settlement time
+
+There are economic constraints that must not be violated:
+
+1. Does not alter the total supply or circulating supply of any token.
+
+## Solution components
+
+In summary, tokens that traverse the bridge are locked on the origin chain and then wrapped tokens are minted and emitted on the destination chain. A wrapped token on chain B is collateralized claim on an actual token on chain A. In reverse, wrapped tokens are redeemed and destroyed, which causes the bridge to send a signal to the other chain that causes an equal amount of collateral to be released. 
+
+<!-- TODO: insert flow diagram --->
+
+1. On-chain logic: Can be a Cosmos SDK Module or a smart contract (depending on which chain the component runs on). Each chain needs a component that can lock tokens in an escrow-like container and send a message to the bridge. In the reverse, it needs to receive instructions and release collateral. 
+2. The Oracle: An Oracle is a highly-available logical singleton. High-availability implies that the *implementation* usually consists of many redundant components. An Oracle must follow a strict protocol that strongly discourages misbehaviour. After all, an Oracle malfunction could dilute the supply of a token with serious, deleterious effects on projects exposed to this risk. 
+
+Together, the two sides of the on-chain logic and the Oracle form a bridge. Prior to the Inter-blockchain Protocol (IBC) it was not possible to generalize a bridge solution that would connect any chain to any other chain or allow assets to flow freely between them. While non-Cosmos chains do indeed require customized implementation in order to become compatible, Cosmos' bridge solution presents the architecture to make this possible. The Gravity bridge is one such implementation that connects Ethereum ERC20 tokens with the Cosmos blockchains. 
```

Comment:
"The Gravity bridge is one such implementation that connects Ethereum ERC20 tokens with the Cosmos blockchains. " --> Is this a non-Cosmos SDK chain connecting to Cosmos chains or Cosmos chains connecting to one another? Please specify to make sure this is 100% clear.

Reply:
Also: Does it really connect **ERC-20 tokens** with  Cosmos blockchains? This would mean tokens with a chain and not multiple chains. And: what is meant by "Cosmos blockchains"? Please specify.